frontend/src/metabase/ui/components/inputs/Input/Input.styled.tsx

```diff
@@ -16,10 +16,14 @@ export const getInputOverrides = (): MantineThemeOverride["components"] => ({
     defaultProps: {
       size: "md",
     },
-    styles: (theme, { multiline }: InputStylesParams, { size = "md" }) => ({
+    styles: (
+      theme,
+      { multiline, radius }: InputStylesParams,
```

Comment:
These components really should be respecting any user-added parameters that are passed in above the default styles